composer.json

```diff
@@ -2,18 +2,18 @@
     "name": "silverstripe/demo.silverstripe.org",
     "description": "Demo SilverStripe Project",
     "require": {
-        "php": ">=7.1.0",
-        "silverstripe/installer": "^4.7.0",
-        "silverstripe/dynamodb": "^4.0",
-        "silverstripe/crontask": "*@dev"
+        "php": ">=7.4.0",
+        "silverstripe/installer": "^4.11",
+        "silverstripe/dynamodb": "^4.1",
+        "silverstripe/crontask": "^2.5"
     },
     "require-dev": {
-        "phpunit/phpunit": "^5.7"
+        "phpunit/phpunit": "^9.5"
```

Comment:
I'd be inclined to just remove this - the demo site has no tests.

Reply:
Removed